coderd/reaper/detector.go

```diff
@@ -371,3 +395,13 @@ func unhangJob(ctx context.Context, log slog.Logger, db database.Store, pub pubs
 
 	return nil
 }
+
+func reapParamsFromJob(job database.ProvisionerJob) (jobType, float64) {
```

Comment:
The returned threshold should just be a `time.Duration`. If you want to print it as minutes in the string you could call `.Minutes()` on it in the format call